cmd/log-parser/README.md

```diff
@@ -0,0 +1,94 @@
+# `kata-log-parser`
+
+* [Logfile requirements](#logfile-requirements)
+* [Component logfiles](#component-logfiles)
+* [Usage](#usage)
+
+`kata-log-parser` is a tool that combines logfiles generated by the various
+system components, sorts them by timestamp and re-displays the log entries,
+adding a time delta showing how much time has elapsed between each log entry.
+
+The tool also checks the validity of all log records and can re-format the
+logs and output them in a different format.
+
+For full details,
+
+```
+$ kata-log-parser --help
+```
+
+## Logfile requirements
+
+The tool reads logfiles in the [logfmt](https://brandur.org/logfmt) structured
+logging format, for example as created by the golang
```

Comment:
Lines 23-24 suggested rewrite: logging format. For example, a logfile created by the golang [logrus](https://godoc.org/github.com/sirupsen/logrus) package.